### PR TITLE
p1-b1-r32 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-de9144/cod-bxxqqq_bHY6yh-de9144.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        <pb ed="#B" n="15-v"/>
+        <cb ed="#B" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-de9144"><!-- p1l1-r32 -->
+        <head xml:id="bHY6yh-de9144-Hd1e108">Rubrica 32</head>
+        <head xml:id="bHY6yh-de9144-Hd1e111" type="question-title">De christo et trinitate diuina</head>
+        <p xml:id="bHY6yh-de9144-d1e114">
+          De christo et trinitate diuina Rubrica 32
```
Comment (jeffreycwitt) [inline bHY6yh-de9144/cod-bxxqqq_bHY6yh-de9144.xml:64]: are we missing a line break here?

**Review by jeffreycwitt — APPROVED** [review]
I think there is a missing line break. If there is please fix, otherwise just confirm that there is no line break here and I will merge.

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
need confirmation whether there is a missing line break at the beginning or not.

**Comment by GrantSchreib** [comment]
corrected

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**